contracts/implementation/AutoClaim.sol

```diff
@@ -162,6 +162,14 @@ contract AutoClaim is IAutoClaim, Initializable {
         return request.updateIntervalId > 0 && request.updateIntervalId < currentUpdateIntervalId;
     }
 
+    receive() external payable {
```

Comment:
why did removing function fix the lint error? Does it not need the function keyword here?

Reply:
I guess not -- I can track down some documentation if you'd like?

Reply:
https://blog.soliditylang.org/2020/03/26/fallback-receive-split/

example here didn't have function keyword either